src/modules/users/UsersPage.cpp

```diff
@@ -162,6 +162,34 @@ UsersPage::UsersPage( Config* config, QWidget* parent )
             config, &Config::requireStrongPasswordsChanged, ui->checkBoxRequireStrongPassword, &QCheckBox::setChecked );
     }
 
+    // Active Directory is not checked or enabled by default
+    ui->useADCheckbox->setVisible(m_config->getActiveDirectoryEnabled());
+    ui->domainLabel->setVisible(false);
```

Comment:
This collection of `setVisible()` as well as the one connected to `toggled()` would be simplified if you put the collection of widgets that is relevant for AD into a parent widget (a `QFrame` for instance) so you can show / hide the parent widget.